test/e2e/components/vl-alert.js

```diff
@@ -33,8 +29,9 @@ class VlAlert extends VlElement {
         return this._innerText('#title');
     }
 
-    async getMessage() {
-        return this._innerText('#message');
+    async getMessages() {
+        const slot = await this.shadowRoot.findElement(By.css("#messages-slot"));
+        return this.getAssignedElements(slot);
```

Comment:
ik zou hier iets met slot in de naamgeving altijd zetten, om aan de afnemer aan te geven wat hij hier kan verwachten ...